p2p/protocol/handler.go

```diff
@@ -41,7 +41,7 @@ func QuaiProtocolHandler(stream network.Stream, node QuaiP2PNode) {
 			continue
 		}
 
-		go handleMessage(data, stream, node)
+		handleMessage(data, stream, node)
```

Comment:
If a message is received faster than a message is processed, we have introduced a goroutine leak. If you would like, we can create a queue of messages that have to be handled, but that queue will still have to be capped to prevent a memory leak.

Reply:
Why do you consider that a leak? It may be bursts of messages that come in every once in a while that take a little while to process, but imo the recipient should be responsible for queuing those because we can't guarantee that other clients will wait for our processing time.

Reply:
I put the message handling back in a goroutine so it's async, but I will make a separate PR to change it to a worker pool with a limit on the number of goroutines